### ISSUE TITLE
Turn on soft-wrap by default for long one-line file

### ISSUE THREAD

**Issue by ndtoan96** [helix-editor/helix#8184]
When opening a long one-line file (bundled javascript, non-pretty json,...), the helix editor becomes very laggy to the point of unusable. This can be solved by turning on the soft-wrap so the text is span on multiple lines.

Hence, I suggest that we detect if there is a line too long in the file and turn on soft-wrap by default.

**Comment by pascalkuthe** [comment]
yeah I already planned to add a mechanism like that. The problem is that this requires enabling softwrab just for a single file. This again would require the concept of per-buffer settings (I don't want to add another special case here)

**Comment by sprime-team** [comment]
Is this feature came yet? I cant find this on helix documentation.

**Comment by the-mikedavis** [comment]
Not yet - this issue is tracking the feature so it will be closed when the feature is added.
